test/real-person/real-person.key

```diff
@@ -0,0 +1,19 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
This is just some random public key I made to attempt importing

Reply:
I made is 1024 bytes to try to keep it small

Reply:
thanks that's cool 👍